recon/modules/v1_osint/github_user.py

```
@@ -0,0 +1,42 @@
+    description = "Fetch basic GitHub user profile by username."
+
+    def defaults(self) -> Dict[str, Any]:
+        return {}
+
+    def examples(self):
+        return [{"username": "torvalds"}]
+
+    def options(self):
+        return ["username"]
+
+    def run(self, username: str, http: HTTPClient, **kwargs) -> Dict[str, Any]:
```

Ruff (0.13.2)

Unused method argument: `kwargs`

(ARG002)